db/historydb/views.go

```diff
@@ -12,6 +12,80 @@ import (
+	TokenUSDUpdate   *time.Time        `json:"fiatUpdate"`
+}
+
+type exitAPIJSON struct {
+	ItemID                 uint64                          `json:"itemId"`
+	BatchNum               common.BatchNum                 `json:"batchNum"`
+	AccountIdx             apitypes.HezIdx                 `json:"accountIndex"`
+	Bjj                    *apitypes.HezBJJ                `json:"bjj"`
+	EthAddr                *apitypes.HezEthAddr            `json:"hezEthereumAddress"`
+	MerkleProof            *merkletree.CircomVerifierProof `json:"merkleProof"`
+	Balance                apitypes.BigIntStr              `json:"balance"`
+	InstantWithdrawn       *int64                          `json:"instantWithdraw"`
+	DelayedWithdrawRequest *int64                          `json:"delayedWithdrawRequest"`
+	DelayedWithdrawn       *int64                          `json:"delayedWithdraw"`
+	TokenJSON              tokenJSON                       `json:"token"`
+}
+
+type accountAPIJSON struct {
+	ItemID             uint64              `json:"itemId"`
+	AccountIndex       apitypes.HezIdx     `json:"accountIndex"`
+	Nonce              common.Nonce        `json:"nonce"`
+	Balance            *apitypes.BigIntStr `json:"balance"`
+	Bjj                apitypes.HezBJJ     `json:"bjj"`
+	HezEthereumAddress apitypes.HezEthAddr `json:"hezEthereumAddress"`
+	TokenJSON          tokenJSON           `json:"token"`
+}
+
+type txAPIJSON struct {
+	TxID        common.TxID          `json:"id"`
+	ItemID      uint64               `json:"itemId"`
+	Type        common.TxType        `json:"type"`
+	Position    int                  `json:"position"`
+	FromIdx     *apitypes.HezIdx     `json:"fromAccountIndex"`
+	FromEthAddr *apitypes.HezEthAddr `json:"fromHezEthereumAddress"`
+	FromBJJ     *apitypes.HezBJJ     `json:"fromBJJ"`
+	ToIdx       apitypes.HezIdx      `json:"toAccountIndex"`
+	ToEthAddr   *apitypes.HezEthAddr `json:"toHezEthereumAddress"`
+	ToBJJ       *apitypes.HezBJJ     `json:"toBJJ"`
+	Amount      apitypes.BigIntStr   `json:"amount"`
+	BatchNum    *common.BatchNum     `json:"batchNum"`
+	HistoricUSD *float64             `json:"historicUSD"`
+	Timestamp   time.Time            `json:"timestamp"`
+	L1Info      *l1infoJSON          `json:"L1Info"`
+	L2Info      *l2infoJSON          `json:"L2Info"`
+	TokenJSON   tokenJSON            `json:"token"`
+	L1orL2      string               `json:"L1orL2"`
+}
+
+type l1infoJSON struct {
+	ToForgeL1TxsNum          *int64              `json:"toForgeL1TransactionsNum"`
+	UserOrigin               *bool               `json:"userOrigin"`
+	DepositAmount            *apitypes.BigIntStr `json:"depositAmount"`
+	AmountSuccess            bool                `json:"amountSuccess"`
+	DepositAmountSuccess     bool                `json:"depositAmountSuccess"`
+	HistoricDepositAmountUSD *float64            `json:"historicDepositAmountUSD"`
+	EthereumBlockNum         int64               `json:"ethereumBlockNum"`
+}
+
+type l2infoJSON struct {
```

Comment:
@mfcastellani , could you make these new structs public? This way we can also reuse them in SDK or external libraries.